README.md

```diff
@@ -652,6 +653,51 @@ The supported Picture in Picture events on `PlayerView` are:
 
 Check [`events.ts`](https://github.com/bitmovin/bitmovin-player-react-native/blob/development/src/components/PlayerView/events.ts) for more information about them.
 
+### Customize HTML UI (Android and iOS only)
+
+The Bitmovin Player SDKs use the open source [Bitmovin Player Web UI](https://github.com/bitmovin/bitmovin-player-ui) on all platforms, except tvOS.
+This is cusomizable in multiple ways.
```

Comment:
NIT:
For me the sentence does not make it 100% clear what "this" is referring to.
```suggestion
The UI is cusomizable in multiple ways.
```

Reply:
fixed in 84595e0